### ISSUE TITLE
Possible script error browser

### ISSUE THREAD

**Issue by mitm001** [SoftbearStudios/mk48#206]
The situation is that on occasion the game will lock you out from selecting buttons like upgrading ships.

The two times I had it occur were over these last two days. 

Torpedoes and decoys also start acting funny like in my previous posts on the subject where my decoys get ignored but opponents become super duper strong as in way out of range and behind my sub nose yet torpedoes go right through opponent to them, they make extreme turns like U turns at point blank range, torpedoes lock onto target and then unlock and wander off into the world somewhere when no decoys or ships around or all but one torpedo or missile hits target and the last fired just goes through target. Even missiles are affected as in they may go opposite direction like 180 behind rather than forward.

May be a script interfering not as in cheat script but like somethings running infinitely causing other scripts not to run.

It takes a refresh to fix it. 

Other player reported he got word null when firing torpedoes. Not sure what that was on so I will ask him to clarify. 

Win 7, Firefox browser with private browsing and clear history settings on close.

**Comment by finnbear** [comment]
> The situation is that on occasion the game will lock you out from selecting buttons like upgrading ships.

We are in the process of rebuilding the entire GUI from the ground up, so this may alleviate some bugs due to Svelte mis-reconciliation of GUI elements.

>  where my decoys get ignored

Decoys work *much* better when they are approximately the same distance to the enemy homing torpedoes as you are and you disable active sensors.

> May be a script interfering not as in cheat script but like somethings running infinitely causing other scripts not to run.
> It takes a refresh to fix it.

That could explain GUI issues

> Other player reported he got word null when firing torpedoes. Not sure what that was on so I will ask him to clarify.

:eyes:

**Comment by mitm001** [comment]
> distance to the enemy homing torpedoes as you are and you disable active sensors.

Yes, I know, this all started again over the last 2 days. I don't think you did an update lately so maybe this will get fixed when done like you said.

The improvements have been working well until last 2 days when all of this started happening. I even broke my own record last night for attacking a high level fleet and surviving. Beat 4 players and almost got the last 2 a few minutes later but they were beat up pretty bad before they got me. Their very bad base design helped out alot once I broke into it.
